pkg/controller/compliancescan/scan.go

```diff
@@ -311,6 +312,9 @@ func (r *ReconcileComplianceScan) newPlatformScanPod(scanInstance *compv1alpha1.
 		"--resultdir=" + PlatformScanDataRoot,
 		"--profile=" + scanInstance.Spec.Profile,
 		"--warnings-output-file=/reports/warning_output",
+		"--namespace=" + scanInstance.Namespace,
+		"--pod-name=" + podName,
```

Comment:
For the namespace and pod name, have you considered using a downward API instead? See https://kubernetes.io/docs/concepts/workloads/pods/downward-api/

Reply:
nice, I didn't know they were set as env variable